src/components/ScrollPage.tsx

```diff
@@ -0,0 +1,149 @@
+
+    const { height, width } = Dimensions.get("window")
+
+    const { articleTitle, articleSubtitle } = usePalette()
+    return (
+        <View style={{ flex: 1 }}>
+            {isLight && (
+                <LinearGradient
+                    colors={[
+                        "#424967",
+                        "rgba(66, 73, 103, 0.43)",
+                        "rgba(66, 73, 103, 0.252403)",
+                        "rgba(66, 73, 103, 0)",
+                    ]}
+                    style={{
+                        flex: 1,
+                        zIndex: 1,
+                        width: width,
+                        height: height,
+                        position: "absolute",
+                    }}
+                    locations={[0.04, 0.19, 0.55, 0.96]}
+                ></LinearGradient>
+            )}
+            {!isLight && (
+                <View
+                    style={{
+                        zIndex: 1,
+                        backgroundColor: "rgba(27, 33, 50, 0.63)",
+                        flex: 1,
+                        position: "absolute",
+                        top: 0,
+                        width: width,
+                        height: 230,
+                    }}
+                ></View>
+            )}
+            {props.backdropElement && (
+                <View
+                    style={{
+                        height: 230,
+                        width: "100%",
+                        position: "absolute",
+                        top: 0,
+                        zIndex: 0,
+                    }}
+                >
+                    {props.backdropElement}
+                </View>
+            )}
+
+            <View
+                style={{
+                    backgroundColor: background,
+                    borderTopLeftRadius: 30,
+                    borderTopRightRadius: 30,
+                    marginTop: 192,
+                    paddingHorizontal: 24,
+                    paddingTop: 8,
+                    zIndex: 2,
+                }}
+            >
+                <ScrollView
+                    showsVerticalScrollIndicator={false}
+                    refreshControl={
+                        props.refreshControl ? (
+                            <RefreshControl
+                                refreshing={props.refreshControl.refreshing}
+                                onRefresh={props.refreshControl.onRefresh}
+                            />
+                        ) : undefined
+                    }
+                >
+                    <View style={{ paddingHorizontal: 4 }}>
+                        <Title
+                            style={{
+                                fontSize: 42,
+                                fontFamily: "Roboto_900Black",
+                                color: articleTitle,
+                            }}
+                        >
+                            {props.title}
+                        </Title>
```

Comment:
in qualche modo il titolo è troppo in alto, non so se va aggiunto un padding al wrapper, un margine al titolo o un inset alla scrollview, ma la scritta è troppo vicina alla parte superiore rispetto a com'è su figma

<img width="424" alt="image" src="https://user-images.githubusercontent.com/34661230/206196018-76ec656a-5fff-4f89-bfd9-7036d879d830.jpeg">
<img width="424" alt="image" src="https://user-images.githubusercontent.com/34661230/206196093-f3042cb7-e606-4971-a5b2-b9bc14c77e1b.png">

Reply:
Ok, ho aggiunto anche qualche pixel di padding sotto il sottotitolo:
![Screenshot_2022-12-07-16-38-48-731_host.exp.exponent.jpg](https://user-images.githubusercontent.com/43524162/206223634-ce0f3d25-fa2a-4c4c-a900-0e4bc2554005.jpg)